iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -365,6 +381,8 @@ public static void checkConvertersMode(Validable validable, HvdcLine.ConvertersM
     public static void checkPowerFactor(Validable validable, double powerFactor) {
         if (Double.isNaN(powerFactor)) {
             throw new ValidationException(validable, "power factor is invalid");
+        } else if (Math.abs(powerFactor) > 1) {
+            throw new ValidationException(validable, "power factor is invalid, it should be between -1 and 1");
```

Comment:
If the power factor is between -1 and 1, we should also check the lower bound

Reply:
I cannot make it with Math.abs ?

Reply:
You right

Reply:
I prefer to use  the standard definition of power factor `P/sqrt(P*P + Q*Q)`. You can check the wikipedia definition (https://en.wikipedia.org/wiki/Power_factor).

In our particular case the active power `P` is always positive so the power factor will be in the interval [0,1] but I prefer to check it between -1 and 1 as we already have an additional check for the active power to be positive.  This is my particular opinion.

Reply:
Finally, do you agree with this part of the code ? I "accidentaly" merge this PR minutes ago…

Reply:
Yes, I've just finished to review it, it is okay for me, we can leave it merged as it is!

Reply:
Totally agree.